src/couch_replicator/src/couch_multidb_changes.erl

```diff
@@ -0,0 +1,819 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License. You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+-module(couch_multidb_changes).
+-behaviour(gen_server).
+
+-export([start_link/4]).
+
+-export([init/1, handle_call/3, handle_info/2, handle_cast/2]).
+-export([code_change/3, terminate/2]).
```

Comment:
I did some cleanups in this area. Restructured exports, fixed some > 80 line chars.  I am sure didn't get all of them. 

I tried using https://github.com/inaka/elvis with a custom style config. But the number of rules there are not enough and some are not useful. 80 char lies was probably the most helpful one, others I had to ignore to avoid generating noise.

I think we need a customized emacs-based reformatter thing like this: https://github.com/fenollp/erlang-formatter. 

Also, some of the code was copied from old modules, for example couch_scheduler_scheduler_job is mostly the old couch_replicator gen_server I skipped some of those to not make the diff even larger. 

We'd probably do a reformatting pass-only as a separate set of PRs and such. But it would be good to have an automated formatter-thing first.